phpcs.xml.dist

```diff
@@ -3,6 +3,7 @@
 	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
 	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards -->
 	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki -->
+	<!-- See https://github.com/WPTRT/WPThemeReview/ -->
 	<!-- See https://github.com/wimg/PHPCompatibility -->
```

Comment:
Comments for all the other coding standards should probably be removed since the project will only rely on `WPThemeReview` then.

Reply:
I'm ok with leaving them since `WPThemeReview` depends on both `WPCS` and `PHPCompatibility` standards 🙂